flask_cors/__init__.py

```diff
@@ -298,6 +312,19 @@ def _reg_length(maybe_regex):
     else:
         raise ValueError("Unexpected value for resources argument.")
 
+def _stringify_regexp(regexp):
+    '''
+        Helper that returns regexp pattern from given value.
+
+        :param regexp: regular expression to stringify
+        :type regexp: _sre.SRE_Pattern or str
+        :returns: string representation of given regexp pattern
+        :rtype: str
+    '''
+    try:
+        return regexp.pattern
+    except AttributeError:
+        return str(regexp)

```

Comment:
I've changes `_stringify_regexp()` to `_get_regexp_pattern()`. However I'm in favour of more pythonic "ask for forgiveness" rather than checking for type.